frontend/src/app/main/ui/components/radio_buttons.cljs

```diff
@@ -0,0 +1,77 @@
+;; This Source Code Form is subject to the terms of the Mozilla Public
+;; License, v. 2.0. If a copy of the MPL was not distributed with this
+;; file, You can obtain one at http://mozilla.org/MPL/2.0/.
+;;
+;; Copyright (c) KALEIDOS INC
+
+(ns app.main.ui.components.radio-buttons
+  (:require-macros [app.main.style :refer [css]])
+  (:require
+   [app.common.math :as math]
+   [app.util.dom :as dom]
+   [rumext.v2 :as mf]))
+
+(def ctx-radio-button (mf/create-context nil))
```

Comment:
Just a comment, i like how you solved this with the context. I think it's a good way for this use case :+1: